### PR TITLE
update hydrogen pipeline cost data to new EHB numbers

### LINKED ISSUE #115: update H2 pipeline costs for inflation
see https://www.hydrogeninsight.com/production/europes-hydrogen-backbone-of-cross-border-pipelines-will-cost-billions-more-euros-than-initial-estimates/2-1-1560429

### PR DESCRIPTION
**Pull request by fneum**
closes #115 

source https://ehb.eu/files/downloads/EHB-2023-20-Nov-FINAL-design.pdf

### CODE REVIEW

**Comment by lisazeyen** [comment]
@fneum you updated the compressor costs as well from 3.4 MEUR/MWe to 4 MEUR/MWe for compressor. Is the source for the compressor costs and the named Table still up to date? -> European Hydrogen Backbone Report, Table 35

**Comment by fneum** [comment]
Ah, no, it's also from the new report.

File: inputs/manual_input.csv [hunk 1]
```diff
@@ -55,12 +55,18 @@ CH4 (g) submarine pipeline,investment,2002,112.64,EUR/MW/km,2014,Kaiser (2017):
...
+H2 (g) submarine pipeline repurposed,FOM,2015,3,%/year,2015,Assume same as for CH4 (g) submarine pipeline.,-
+H2 (g) submarine pipeline repurposed,lifetime,2015,30,years,2015,Assume same as for CH4 (g) submarine pipeline.,-
+H2 (g) submarine pipeline repurposed,electricity-input,2020,0.021,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
+H2 (g) submarine pipeline repurposed,electricity-input,2030,0.019,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
+H2 (g) submarine pipeline repurposed,electricity-input,2050,0.017,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
```
Comment (euronion) [inline inputs/manual_input.csv:69]: In principle we could align these (and for other H2 pipeline technologies) also with the EHB:

434 MW_e / 16900 MW_H2 / 1000 km = 0.0257 MW_e/MW_H2 / 1000km

Very similar number. What do you thinkg

**Review by euronion — CHANGES_REQUESTED** [review]
Mostly looking good, a few comments

File: inputs/manual_input.csv (hunk 1 above)
Comment (fneum) [reply inputs/manual_input.csv:69]: Let's keep the way it is.

**Comment by euronion** [comment]
LGTM

**Review by euronion — APPROVED** [review]

**Pull request merged**